File: demo/module/src/main/java/com/timper/lib/di/TestRepository.java
package com.timper.lib.di;

import com.timper.lonelysword.annotations.apt.UseCase;
import io.reactivex.Flowable;

/**
 * User: tangpeng.yang
 * Date: 2019/3/20
 * Description:
 * FIXME
 */
public interface TestRepository {

    @UseCase
    Flowable<String> getUseCase();
}

File: demo/module/src/main/java/com/timper/module/data/remote/TestRepository.kt
package com.timper.module.data.remote

import io.reactivex.Flowable

/**
 * User: tangpeng.yang
 * Date: 2019/3/20
 * Description:
 * FIXME
 */
interface TestRepository {

    fun getUseCase(): Flowable<String>

}
